@mzm/core/resource/types.ts

```diff
@@ -6,6 +6,19 @@ export enum StringifyFormat {
 
 export interface IResourceDefinition {
   type: string
-  spec: object
+  spec: Record<string, unknown>
 }
 
+export interface IResourceTemplate {
+  version: 'v1' | 'v2' | 'v3'
```

Comment:
I've only seen implementations for v1 and v3.
Support for view and category are not provided in v3.
What am I missing?